creator-node/src/services/sync/syncUtil.ts

```diff
@@ -138,5 +177,33 @@ async function fetchExportFromNode({
   }
 }
 
-export { fetchExportFromNode }
-export type { FetchExportParams, FetchExportOutput }
+async function getSyncStatus(syncUuid: string) {
+  return redisClient.get(`syncStatus${syncUuid}`) as Promise<SyncStatus | null>
```

Comment:
not sure if one is more correct technically, but because redisClient returns string instead SyncStatus then we have to cast it either way. and with the casting the type system infers the return type so adding the return type would be redundant

Reply:
coo